### PR TITLE
Changing the way the search result is shown

### PR DESCRIPTION
**Pull request by raonismaneoto**
**Feature/Bug description:**
The way the search's result was being shown wasn't matching with the designer's specification

**Solution:**

screenshot from 2019-02-28 10-34-49

screenshot from 2019-02-28 10-35-10

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/search/search_mobile.html [hunk 1]
```diff
@@ -11,41 +11,57 @@
...
-    <div layout="row" layout-align="end center" class="search-mobile-action-buttons">
-        <md-button md-colors="{background:'grey-500'}" ng-click="searchCtrl.leaveMobileSearchPage()">
-            CANCELAR
-        </md-button>
-        <md-button md-colors="{background:'teal-500'}" ng-click="searchCtrl.search()"
-            ng-disabled="!searchCtrl.hasChanges">
-            PESQUISAR
-        </md-button>
+    <div ng-if="!searchCtrl.hasNotSearched">
+        <div class="registered-institutions-container" ng-if="searchCtrl.institutions.length > 1" style="margin-top: -1px; margin-bottom: 5px;">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: move style

File: frontend/search/search_mobile.html [hunk 2]
```diff
@@ -11,41 +11,57 @@
...
+            <registered-institution ng-repeat="institution in institutions = (searchCtrl.institutions 
+                        | filter: searchCtrl.searchActuation | filter: searchCtrl.searchNature | filter: searchCtrl.searchState)" 
+                institution="institution" user="searchCtrl.user" style="margin-top: 10px;">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: move style

File: frontend/search/search_mobile.html [hunk 3]
```diff
@@ -11,41 +11,57 @@
...
+            </registered-institution>
+        </div>
+        <div ng-if="searchCtrl.institutions.length === 1">
+            <registered-institution institution="searchCtrl.institutions[0]" 
+                user="searchCtrl.user" style="border-style: solid; border-width: 0.5px 0px 0px 0.5px; border-color: white;">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: move style

File: frontend/search/search_mobile.html [hunk 4]
```diff
@@ -11,41 +11,57 @@
...
+            </registered-institution>
+        </div>
+        <div ng-if="searchCtrl.institutions.length === 0" style="margin-top: 1em; margin-left: auto; margin-right: auto;">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: move style

Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: use empty card component

File: frontend/search/search_mobile.html [hunk 5]
```diff
@@ -11,41 +11,57 @@
             </md-input-container>
         </form>
     </div>
-    <div class="search-mobile-options-title">
+    <div class="search-mobile-options-title" id="search-title">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: font-size bigger than it should

Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: margin top and bottom should be the same

image

File: frontend/search/search_mobile.html [hunk 6]
```diff
@@ -11,41 +11,57 @@
...
-            PESQUISAR
-        </md-button>
+    <div ng-if="!searchCtrl.hasNotSearched">
+        <div class="registered-institutions-container" ng-if="searchCtrl.institutions.length > 1" style="margin-top: -1px; margin-bottom: 5px;">
+            <registered-institution ng-repeat="institution in institutions = (searchCtrl.institutions 
+                        | filter: searchCtrl.searchActuation | filter: searchCtrl.searchNature | filter: searchCtrl.searchState)" 
+                institution="institution" user="searchCtrl.user" style="margin-top: 10px;">
+            </registered-institution>
+        </div>
+        <div ng-if="searchCtrl.institutions.length === 1">
```
Comment (JuliePessoa) [inline frontend/search/search_mobile.html]: Margin-left and rigth of the card content is bigger than it should.

image

**Review by JuliePessoa — CHANGES_REQUESTED** [review]
- Result institutions are being marked "Novo cadastro"
- Address and image is not showing
- Try to fix the icon resize problem

**Review by JuliePessoa — APPROVED** [review]

**Review by MaianaB — APPROVED** [review]

**Pull request merged**